Fix several issues in webext storage docs (#43924)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/storage/storagearea/onchanged/index.md

sidebar: addonsidebar
---

> Fires when one or more items in a storage area change, with details for the keys that changed. Compared to {{WebExtAPIRef("storage.onChanged")}}, this event enables you to listen for changes in one of the storage areas: `local`, `managed`, `session`, and `sync`.

Reviewer: ```suggestion
Fires when one or more items in a storage area change. Compared to {{WebExtAPIRef("storage.onChanged")}}, this event enables you to listen for changes in one of the storage areas: `local`, `managed`, `session`, or `sync`.
```

> [!NOTE]

> > In Firefox, the listener receives all keys within the storage area {{WebExtAPIRef('storage.StorageArea.set','storageArea.set')}} ran against whether they changed or not. The listener may be invoked when there is no change to the underlying data. Details of the changed items are found by examining each key's {{WebExtAPIRef('storage.StorageChange')}} object. See [Firefox bug 1833153](https://bugzil.la/1833153).

Reviewer: ```suggestion
> In Firefox, the listener receives all the keys from a storage area where {{WebExtAPIRef('storage.StorageArea.set','storageArea.set')}} executes. The listener may also be invoked when there is no change to the data. To find details of the changed items, examine each key's {{WebExtAPIRef('storage.StorageChange')}} object. See [Firefox bug 1833153](https://bugzil.la/1833153).
```
